### DISCUSSION TITLE
Automatically hide inactive profiles

### DISCUSSION THREAD

**Discussion by Oshyan** [PullDating/Org#124] [Ideas & Features]
One big problem with many dating apps (Hinge for sure, OKC, probably others) is you have no idea when they last logged-in to the app. Many people forget to or just don't care to manually deactivate their profiles if they meet someone or stop looking. This can partly be addressed by #79 , however I also think that after a certain longer period of inactivity (3 months? maybe less?), inactive profiles should actually be fully hidden from searches, matching, and at the very least marked in someone's messages/match queue if they already had matched with them prior to them going inactive. I think this is a key part of us respecting the time and energy of all users and helping them to focus on matches with actual potential. In fact the more I think about it, the more I think a 1 month limit is probably long enough. And I might even consider a little colored indicator to show "user hasn't been online in the last week" or something, not to actually hide those profiles, but just to give people a heads-up.

Most traditional dating apps have little incentive to hide any profiles for any reason. They want pure numbers. We want numbers too, but *legitimate* ones. It's going to be a challenge because we can't/won't pad our number of users with inactive ones (though we can quote total signups vs. MAU). But it's a pill we have to swallow to be transparent and ethical, which is paramount.

**Comment by WilliamDormer** [comment]
This is going to be super important. Since we don't rely on the illusion of the platform being active, we can be very tough on this such as removing them after 72 hours of inactivity. I think people that are actively using dating apps check them a lot more frequently than once a month, so waiting that long probably isn't necessary.

**Reply by Oshyan** [reply]
I agree about less than 1 month, but 72hrs seems rather harsh 😄 Let's say 1 week for now, maybe 2 weeks at most.
